cves/CVE-2006-5752.yml

```diff
@@ -168,8 +180,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: yes
+    note: xss vulnerability
```

Comment:
Could you expand on how this applies, more than just writing "xss vulnerability"?

Reply:
expanded note.